### DISCUSSION TITLE
Is OKL faster than OpenMP version of the code?

### DISCUSSION THREAD

**Discussion by eswarnitk** [libocca/occa#563] [Q&A]
I have created JIT openmp and JIT okl version of the same code(kernel). In my case openmp code runs faster than OKL version, is that  true?

Can anyone explain me what is OCCA-OpenMP and OCCA-OKL?
If possible which version runs faster?

Thanks in advance.

**Comment by tcew** [comment]
You can inspect the kernel source code that OCCA creates. Run with the environment variable OCCA_VERBOSE=1 . It will show you all kernel compilations and the location of the generated source files in the OCCA cache directory.
